packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -0,0 +1,627 @@
+# Cart and wishlist
+
+> This page assumes you're familiar with Composition API and Composables. Read them first if you are new to these concepts.
```

Comment:
I'm not sure what is general approach. Because if we assume that user reads all docs one by one it's okay. But if you assume that someone just looks for certain solution that repetitive info makes sense.